index.html

```diff
@@ -192,6 +192,46 @@ <h2 class="hf">About Me</h2>
           </ul>
         </div>
       </section>
+      <section class="contacts">
+        <p class="hg">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/xayzolrz" class="getcontact" method="post">
+              <input class="input"  type="text" name="name" placeholder="Full name" maxlength="30" required/>
+              <input  class="input" type="email" name="email" placeholder="Email address"  maxlength="30"required>
```

Comment:
- [ ] Kindly consider removing `maxlength="30"` from `email input` because it won't let, a user having an email consisting of more than 30 characters, submit the form. this `maxlength="30"` is required only in the `name input field`.